src/lib/util/contenthandler.js

```diff
@@ -278,6 +278,58 @@ define([
 		}
 	}
 
+	/**
+	 * Transform elements according to mapping set in configuration
+	 * Example config used for fixing Chrome 75 copy paste issues in contenteditable
+	 * Chrome will try to maintain styling on the copied range.
+	 * Depening on the styling of the copied element this may lead to <b> elements turning into <span style="font-size: 700">
+	 * Chrome is unable to copy <sub> and <sup> elements in content editable. This leads to <sub> and <sup> elements turning into spans
+	 * 
+	 *	Aloha.settings.contentHandler.handler.generic.transformFormattingsMapping: [
+	 *		{
+	 *			nodeNameIs: 'span',
+	 *			nodeNameShould: 'b',
+	 *			attribute: {
+	 *				name: 'style',
+	 *				value: 'font-weight: 700'
+	 *			}
+	 *		},
+	 *		{
+	 *			nodeNameIs: 'span',
+	 *			nodeNameShould: 'sup',
+	 *			attributes: {
+	 *				name: 'style',
+	 *				value: 'top: -0.5em'
+	 *			}
+	 *		},
+	 *		{
+	 *			nodeNameIs: 'span',
+	 *			nodeNameShould: 'sub',
+	 *			attributes: {
+	 *				name: 'style',
+	 *				value: 'bottom: -0.5em'
+	 *			}
+	 *		}
+	 *	]
+	 * @param {jQuery.<HTMLElement>} $elem the element
+	 */
+	function transformFormattingsByMapping($elem) {
+		if(isTransformFormattingsByMapping()) {
+			Aloha.settings.contentHandler.handler.generic.transformFormattingsMapping.forEach(function (mapping) {
+				if (typeof mapping.nodeNameIs !== 'undefined' &&
+					typeof mapping.nodeNameShould !== 'undefined' &&
+					typeof mapping.attribute !== 'undefined' &&
+					$elem[0].nodeName.toLowerCase() === mapping.nodeNameIs &&
+					$elem[0].hasAttribute(mapping.attribute.name) &&
```

Comment:
Maybe let `attribute` be an array, in case there are new cases which require more than one to match a mapping?

Reply:
During implementation I found have just a single value is enough to solve the support ticket raised by this issue. The configuration seemed complicated enough at the time. So I will leave it as it is.